nilearn/glm/tests/test_thresholding.py

```diff
@@ -103,6 +104,24 @@ def test_threshold_stats_img():
         threshold_stats_img(None, None, alpha=0.05, height_control='plop')
 
 
+def test_hommel():
+    # Check that the computation of Hommel value
+    # For these, we take the example in  Meijer et al. 2017
+    # 'A shortcut for Hommel's procedure in linearithmic time'
+    # and check that we obtain the same values
+    z = norm.isf([1.e-8, .01, .08, .1, .5, .7, .9])
+    assert(_compute_hommel_value(z, 1.e-9) == 7)
```

Comment:
the parentheses make it look like a function call which is a bit weird, I would remove them or at least add a space between `assert` and the expression